app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -62,4 +62,10 @@ private void startProjectActivity(final @NonNull Project project, final @NonNull
   private void renderProjectAndStats(final @NonNull Pair<Project, ProjectStatsEnvelope> projectAndStats) {
     this.adapter.takeProjectAndStats(projectAndStats);
   }
+
+  @Override
+  protected void onDestroy() {
+    super.onDestroy();
+    creatorDashboardRecyclerView.setAdapter(null);
+  }
```

Comment:
so unsure about whether I need to override `onDestroy` here or in the `RewardStatsViewHolder` below.  I see we are doing it in other activities/viewholders, but when I add or delete it here it doesn't seem to affect behavior, and I don't know if the instance of the adapter gets deallocated when the activity/viewholder gets deallocated.